### PR TITLE
1letter/bs5

### PR DESCRIPTION
**Pull request by 1letter**
Update Page Templates, make it ready for use in Plone 6 with Bootstrap 5

### CODE REVIEW

**Comment by agitator** [comment]
Updated overall markup to align with Plone 6 controlpanels - used https://github.com/plone/plone.app.contentrules as template

Updated translations https://github.com/collective/plone.app.locales/pull/349 probably some more things to look over...

Todo:
- [ ] change template urls from prefs_workflow_localpolicies_form to prefs-workflow-localpolicies-form to align with new controlpanel url schema
- [ ] better naming for prefs_workflow_localpolicies_form, maybe workflow-policies-controlpanel

**Comment by 1letter** [comment]
@agitator if we rename the configlet url, then an upgrade step is needed, because the url of the action in portal_action  changed. should this go to plone.app.upgrade or should we handle this in this package?

**Comment by agitator** [comment]
I'd add the upgrade steps here.
cc @mauritsvanrees

**Comment by mauritsvanrees** [comment]
> I'd add the upgrade steps here.
> cc @mauritsvanrees  

Indeed.

**Comment by 1letter** [comment]
if i adjust the name of browserviews, like suggested by @agitator , a change in [plone.app.contentmenu](https://github.com/plone/plone.app.contentmenu/blob/master/plone/app/contentmenu/menu.py#L913) is needed.

**Comment by 1letter** [comment]
@mauritsvanrees 
if all tests green, this can only merged with this [pr in plone.app.contentmenu](https://github.com/plone/plone.app.contentmenu/pull/42)

File: Products/CMFPlacefulWorkflow/browser/configure.zcml [hunk 1]
```diff
@@ -5,23 +5,23 @@
 
     <browser:page
         for="*"
-        name="placeful_workflow_configuration"
+        name="workflow-policies-controlpanel"
         class=".views.PlacefulWorkflowConfiguration"
         template="placeful_workflow_configuration.pt"
```
Comment (agitator) [inline Products/CMFPlacefulWorkflow/browser/configure.zcml:10]: I'd change the template name to workflow_policies_controlpanel.pt

File: Products/CMFPlacefulWorkflow/browser/configure.zcml [hunk 2]
```diff
@@ -5,23 +5,23 @@
...
         permission="CMFPlacefulWorkflow.ManageWorkflowPolicies"
         />
 
     <browser:page
         for="*"
-        name="prefs_workflow_policy_mapping"
+        name="prefs-workflow-policy-mapping"
```
Comment (agitator) [inline Products/CMFPlacefulWorkflow/browser/configure.zcml:16]: remove 'prefs-' and update template name?

File: Products/CMFPlacefulWorkflow/browser/configure.zcml [hunk 3]
```diff
@@ -5,23 +5,23 @@
...
         class=".views.WorkflowPolicyMapping"
         template="prefs_workflow_policy_mapping.pt"
         permission="CMFPlacefulWorkflow.ManageWorkflowPolicies"
         />
 
     <browser:page
         for="*"
-        name="prefs_workflow_localpolicies_form"
+        name="prefs-workflow-localpolicies-form"
```
Comment (agitator) [inline Products/CMFPlacefulWorkflow/browser/configure.zcml:24]: remove 'prefs-' and update template name?

**Comment by agitator** [comment]
one more thing... view calls should all be done with @@ ... quicker and less acquisition magic ;-)

**Review by agitator — APPROVED** [review]
lgtm

**Pull request merged**